pkgs/servers/nosql/hugegraph/unwrapped.nix

```diff
@@ -0,0 +1,99 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, maven
+, jdk11
+, protobuf
+}:
+
+let
+  pname = "hugegraph-unwrapped";
+  version = "1.0.0";
+  hash = "sha256-NXO/4z1/smKDtJ54LWMUBYAaeQSo9DLE5muZ/nEglvw=";
+  outputHash = "sha256-5b2tH7JV+GZBVhR4hPGhBcffJph3P5djbo9zz/Mx4hc=";
+  fetchedMavenDeps = { pname, src, version, ... }: stdenv.mkDerivation {
+    pname = "${pname}-maven-deps";
+    inherit src version;
+
+    nativeBuildInputs = [
+      (maven.override { jdk = jdk11; })
+      jdk11
+      protobuf
+    ];
+
+    postPatch = ''
+      sed -i 's|<protocArtifact>.*</protocArtifact>|<protocExecutable>${protobuf}/bin/protoc</protocExecutable>|g' hugegraph-core/pom.xml
+      sed -i '/wget /d' hugegraph-dist/pom.xml
+      sed -i '/tar zxvf /d' hugegraph-dist/pom.xml
+      cp -r ${fetchFromGitHub {
+        owner = "swagger-api";
+        repo = "swagger-ui";
+        rev = "v4.15.5";
+        hash = "sha256-cadQhmgOdIGhuH2YCN+6D00KN4vfXrAE1PeyUsC0ppw=";
+      }} hugegraph-dist/swagger-ui-4.15.5
+      chmod +w -R hugegraph-dist/swagger-ui-4.15.5
```

Comment:
This should be moved into a let in and the version number should come from one variable